app/services/representation_order_creator.rb

```diff
@@ -27,10 +27,42 @@ def call_cp_endpoint
         status_date: offence[:status_date],
         effective_start_date: offence[:effective_start_date],
         effective_end_date: offence[:effective_end_date],
-        defence_organisation: defence_organisation
+        defence_organisation: sanitised_defence_organisation
```

Comment:
It could be worth sanitising this in the initialize call so as to have a single memoised instance

Reply:
Moved to `initialize`